doc/developer_guide/psyir_symbols.rst

```diff
@@ -392,9 +392,12 @@ of a PSyIR node or a list of nodes can be gathered by creating an object of
 type `psyclone.core.access_info.VariablesAccessInfo`.
 This class uses a `Signature` object to keep track of the variables used.
 A signature can be thought of as a tuple that consists of the variable name
-and structure members used in an access. For example, an access like
-`a(1)%b(k)%c(i,j)` would be stored with a signature `(a, b, c)`.
-And a simple variable `a` is stored as a one-element tuple `(a, )`.
+and structure members used in an access - called components. For example,
+an access like
+`a(1)%b(k)%c(i,j)` would be stored with a signature `(a, b, c)`, giving
+three components `a`, `b`, and `c`.
+And a simple variable `a` is stored as a one-element tuple `(a, )`, having
```

Comment:
Picky but s/And a simple variable/A simple variable such as/ (to avoid starting a sentence with "and").